backported WebKit changeset 60845

This patch was taken from http://trac.webkit.org/changeset/60845 .
Original commit text is:
"[GTK] Move the declaration of getRootCoords higher up to fix the build on GTK"

For backporting, removed changes to ChangeLog file which apparently doesn't
exist in webkit-stable.

Index: /trunk/WebKitTools/DumpRenderTree/gtk/EventSender.cpp
===================================================================
--- /trunk/WebKitTools/DumpRenderTree/gtk/EventSender.cpp	(revision 60803)
+++ /trunk/WebKitTools/DumpRenderTree/gtk/EventSender.cpp	(revision 60845)
@@ -110,4 +110,19 @@
 }
 
+#if !GTK_CHECK_VERSION(2,17,3)
+static void getRootCoords(GtkWidget* view, int* rootX, int* rootY)
+{
+    GtkWidget* window = gtk_widget_get_toplevel(GTK_WIDGET(view));
+    int tmpX, tmpY;
+
+    gtk_widget_translate_coordinates(view, window, lastMousePositionX, lastMousePositionY, &tmpX, &tmpY);
+
+    gdk_window_get_origin(window->window, rootX, rootY);
+
+    *rootX += tmpX;
+    *rootY += tmpY;
+}
+#endif
+
 bool prepareMouseButtonEvent(GdkEvent* event, int eventSenderButtonNumber)
 {
@@ -177,19 +192,4 @@
         clickCount++;
 }
-
-#if !GTK_CHECK_VERSION(2,17,3)
-static void getRootCoords(GtkWidget* view, int* rootX, int* rootY)
-{
-    GtkWidget* window = gtk_widget_get_toplevel(GTK_WIDGET(view));
-    int tmpX, tmpY;
-
-    gtk_widget_translate_coordinates(view, window, lastMousePositionX, lastMousePositionY, &tmpX, &tmpY);
-
-    gdk_window_get_origin(window->window, rootX, rootY);
-
-    *rootX += tmpX;
-    *rootY += tmpY;
-}
-#endif
 
 static JSValueRef mouseDownCallback(JSContextRef context, JSObjectRef function, JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef* exception)
